google-oauth-client-appengine/pom.xml

```diff
@@ -30,12 +30,12 @@
             <manifest>
               <addDefaultImplementationEntries>true</addDefaultImplementationEntries>
             </manifest>
+            <manifestFile>${project.build.outputDirectory}/META-INF/MANIFEST.MF</manifestFile>
           </archive>
         </configuration>
         <executions>
```

Comment:
Done. What was happening before was that this `<execution>` had a different `<id>jar</id>` than the default `<id>default-jar</id>` (along with `<phase>compile</phase>`), so that's why it was packaging jars twice.